hack/velero/deploy-velero.sh

```diff
@@ -34,6 +35,16 @@ if [[ ! $(_kubectl get deployments -n velero | grep minio) ]]; then
   _kubectl wait -n velero deployment/minio --for=condition=Available --timeout=${DEPLOYMENT_TIMEOUT}s
 fi
 
+if [ ! -f "${velero_path}" ]; then
+  mkdir -p ${velero_dir}
+	echo >&2 "Downloading velero version ${VELERO_VERSION}..."
+	curl -LO https://github.com/vmware-tanzu/velero/releases/download/${VELERO_VERSION}/velero-${VELERO_VERSION}-linux-amd64.tar.gz \
```

Comment:
We need to fetch sources from github anyway, and I have seen this pattern used in some projects that download clusterctl and other binaries. 

But this is actually a good question, as this means, whenever there is a git outage we cannot run tests. 
So maybe I can fetch velero as a docker image from some image hosting service, e.g  `quay.io/konveyor/velero:konveyor-1.8`. That way it will be stored in the image cache that we use in the test infrastructure. But for this I need to check if tests are not using a velero binary directly.

Reply:
You can do 
```make
$(GOBIN)/velero: #install velero if not exist
  Go install GitHub.com/vmware-tanzu/velero/cmd/velero@release-1.9
```

Excuse typos.. mobile

Reply:
@kaovilai thanks for info, I'll try this

Reply:
@kaovilai I did try 
```
> go install github.com/vmware-tanzu/velero/cmd/velero@release-1.9
go: downloading github.com/vmware-tanzu/velero v1.9.2-rc.1
go: github.com/vmware-tanzu/velero/cmd/velero@release-1.9 (in github.com/vmware-tanzu/velero@v1.9.2-rc.1):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.

```

But I cannot find the binary anywhere on my path